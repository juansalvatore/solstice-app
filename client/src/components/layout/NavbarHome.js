import React, { Component } from 'react'
import styled from 'styled-components'

export default class Navbar extends Component {
  render() {
    return (
      <NavbarWrapper>
        <Title>Contacts</Title>
      </NavbarWrapper>
    )
  }
}

const NavbarWrapper = styled.div`
  display: flex;
  height: 60px;
  align-items: flex-end;
  background-color: rgb(249, 249, 249);
  padding: 10px 0;
  font-weight: 500;
  border-bottom: 1px solid rgb(241, 241, 241);
`
const Title = styled.h1`
  width: 100%;
  display: flex;
  justify-content: center;
  font-size: 18px;
`
