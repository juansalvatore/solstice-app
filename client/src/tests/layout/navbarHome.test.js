import React from 'react'
import NavbarHome from '../../components/layout/NavbarHome'
import renderer from 'react-test-renderer'

describe('NavbarHome component', () => {
  it('matches the snapshot', () => {
    const tree = renderer.create(<NavbarHome />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
