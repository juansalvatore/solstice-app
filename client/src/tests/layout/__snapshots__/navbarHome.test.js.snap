// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavbarHome component matches the snapshot 1`] = `
<div
  className="sc-bdVaJa keXJQU"
>
  <h1
    className="sc-bwzfXH dPXpuN"
  >
    Contacts
  </h1>
</div>
`;
